lib/Whim/Core.pm

```diff
@@ -175,10 +177,7 @@ sub fetch_webmentions ( $self, $args ) {
 
             $args{author} = Web::Mention::Author->new( \%author_args );
         }
-        else {
-            $args{author} = undef;
-        }
-        my $wm = Web::Mention->new( \%args );
+        my $wm = Whim::Mention->new( \%args );
```

Comment:
Setting $args{author} to undef was a straight-up mistake - it should be left "clear" so that Moo's can lazily define it